dev-ops/src/main/resources/org/interledger/dev-ops/checkstyle.xml

```diff
@@ -59,7 +59,7 @@
         </module>
         <module name="NeedBraces"/>
         <module name="LeftCurly">
-            <property name="maxLineLength" value="100"/>
+            <!--<property name="maxLineLength" value="100"/>-->
```

Comment:
Inclusion of this property in the checkstyle file breaks the Intellij Checkstyle Plugin. However, if people don't like this, I'm open to restoring the line.

Reply:
That sucks...
Why do we need it?

Reply:
See [here](http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/checks/blocks/LeftCurlyCheck.html):
> Checks the placement of left curly braces on types...

Though I suspect the `maxLineLength` property is not really necessary here, but not sure.